src/modules/ekf2/ekf2_main.cpp

```diff
@@ -165,11 +198,21 @@ class Ekf2 final : public ModuleBase<Ekf2>, public ModuleParams
 	const float _vel_innov_spike_lim = 2.0f * _vel_innov_test_lim;	///< preflight velocity innovation spike limit (m/sec)
 	const float _hgt_innov_spike_lim = 2.0f * _hgt_innov_test_lim;	///< preflight position innovation spike limit (m)
 
+	// GPS blending and switching
+	struct gps_message _gps_state[GPS_MAX_RECEIVERS + 1]; ///< internal state data for the physical and blended GPS
```

Comment:
Suggestion, the code below might be a bit easier to follow if you kept the blended GPS in a separate struct rather than the end.
`_gps_state[GPS_BLENDED_INSTANCE] -> _gps_blended`

Reply:
Agree - wil fix

Reply:
fixed